content/posts/GSoC_Coding_Phase_Blog_4/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "GSoC Coding Phase 2 Blog 2"
+date: 2020-07-23T19:47:51+05:30
+draft: false
+categories: ["News", "GSoC"]
+description: "Progress Report for the second half of the Google Summer of Code 2020 Phase 2 for the Baseline Images Problem"
+displayInList: true
+author: Sidharth Bansal
+---
+
+Google Summer of Code 2020's second evaluation is about to complete. Now we are about to start with the final coding phase. This post discusses about the progress so far in the last two weeks of the second coding period from 13 July to 26 July 2020.
+
+## Modular approach towards removal of matplotlib baseline images
+
+We have divide the work in two parts as discussed in the previous blog. The first part is the generation of the baseline images discussed below. The second part is the modification of the baseline images. Modification part will be implemented last phase of the Google Summer of Code 2020.
+
+## Generation of the matplotlib baseline images
+
+After the changes proposed in the [previous PR](https://github.com/matplotlib/matplotlib/pull/17557), the developer will have no baseline images on fresh install of matplotlib. The developer would need to install the sub-wheel matplotlib_baseline_images package to get started with the testing part of the mpl. Now, we have started removing the use of the matplotlib_baseline_images package. It will require two steps as discussed above.
+The images can be generated by the image comparison tests. Once these images are generated for the first time, then they can be used as the baseline images for the later times for comparison. This is the main principle adopted. The images are first created in the `result_images` directory. Then they are moved to the `lib/matplotlib/tests/baseline_images` directory. Later on, running the pytests will start the image comparison. 
```

Comment:
I thought we were writing them directly to the baseline images now (rather than writing to results and moving)?